proto-compiler/Cargo.lock

```diff
@@ -2,9 +2,9 @@
 # It is not intended for manual editing.
 [[package]]
 name = "anyhow"
-version = "1.0.38"
+version = "1.0.40"
```

Comment:
I am wondering if we should keep the modifications to this file.

Ideally, we would change `proto-compiler/Cargo.lock` only when we re-build the `proto-compiler` binary, and we should rebuild this binary only when we use it to regenerate the .proto file (https://github.com/informalsystems/ibc-rs/tree/master/proto/src). Keeping `proto-compiler` binary aligned with the .proto files allows to regenerate those files in a way that produces consistent results (same Rust output, given same input .proto).

However, [some](https://github.com/informalsystems/ibc-rs/pull/1016) PRs changed the file `proto-compiler/Cargo.lock` anyway since the last time we used it to generate Rust proto files.. and maybe it's just easier to go with the flow and allow changes to this (since we have its history anyway, we should be able to obtain consistent output)

Reply:
As discussed offline, let's keep those in for this release.